app/src/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on subsequent visits to a page, after all the
+// existing tabs open on the page have been closed, since previously cached
+// resources are updated in the background.
+
+// To learn more about the benefits of this model and instructions on how to
+// opt-in, read https://bit.ly/CRA-PWA
```

Comment:
It looks like this code was copied from somewhere. Is it auto-generated by create-react-app?

Reply:
Yes, that's correct. [Here's](https://create-react-app.dev/docs/making-a-progressive-web-app/) the reference to the default serviceWorker.js functionality including its benefits and limitations. 

As a summary, it helps improve caching performance to smoothen the user experience. I wouldn't mind leaving it nor removing it. Wdyt?

Reply:
Is it currently used in the code, or do you have plans to use it for this project? If it's unused code, it should probably just be deleted. Otherwise, it's fine to include here.

Reply:
Yes I thought we could use it since it does help smoothen the user experience quite a lot. But it gets "activated" once you deploy to a production environment.